### DISCUSSION TITLE
希望小狼毫能修改中/英图标

### DISCUSSION THREAD

**Discussion by ideaexplorer233** [rime/home#1122] [Ideas]
我换了个黑暗模式的主题 结果一切换模式就露馅了(
也不能关闭切换时显示(虽说这是必要的 但是我其实可以看托盘图标的 在托盘显示效果很好)
为啥后期不能更改呢
或者不能更改 那颜色跟随一下主题色会好点

**Comment by CDOccc** [comment]
同感，这个图标观感真不太好

**Comment by fxliang** [comment]
https://github.com/rime/weasel/wiki/%E4%B8%BA%E7%89%B9%E5%AE%9A%E8%BE%93%E5%85%A5%E6%96%B9%E6%A1%88%E5%AE%9A%E5%88%B6%E5%A4%96%E8%A7%82#%E8%87%AA%E5%AE%9A%E4%B9%89%E5%9B%BE%E6%A0%87
